install/sct_changelog.py

```diff
@@ -119,18 +119,22 @@ def check_compatibility(dict_labels=''):
         '[View detailed changelog](%s)' % detailed_changelog(title),
     ]
 
+
     changelog_pr = set()
     for label in ['bug', 'enhancement', 'feature', 'documentation', 'installation', 'testing']:
         pulls = search(milestone['title'], label)
         items = pulls.get('items')
         if items:
             lines.append('\n**{}**\n'.format(label.upper()))
             changelog_pr = changelog_pr.union(set([x['html_url'] for x in items]))
-            items = [" - **%s:** %s. %s[View pull request](%s)" % (",".join(get_sct_function_from_label(x['labels'])),
+            for x in pulls.get('items'):
+                items = [" - **%s:** %s. %s[View pull request](%s)" % (",".join(get_sct_function_from_label(x['labels'])),
                                                                 x['title'],
                                                                 check_compatibility(x['labels']),
-                                                                x['html_url']) for x in pulls.get('items')]
-            lines.extend(items)
+                                                                x['html_url'])]
+                if (len(get_sct_function_from_label(x['labels'])) == 0):
+                    items[0] = items[0].replace("**:**","")
```

Comment:
you should replace `"**:**"` by `"**:** "` to avoid the extra space after replacement